library/src/main/java/org/xmtp/android/library/Contacts.kt

```diff
@@ -304,14 +303,14 @@ data class Contacts(
         consentList.publish(entries)
     }
 
-    suspend fun allowInboxId(inboxIds: List<String>) {
+    suspend fun allowInbox(inboxIds: List<String>) {
```

Comment:
```suggestion
    suspend fun allowInboxes(inboxIds: List<String>) {
```

Reply:
Ya this is what I named it in react native. But for some reason didn't make these plurals for any of the other ones (groups and addresses) 😮‍💨